server/src/main/resources/ehcache-stats.xml

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <ehcache
-    maxBytesLocalHeap="300M"
+    maxEntriesLocalHeap="100000"
     >
 
+	<sizeOfPolicy maxDepth="100" maxDepthExceededBehavior="abort"/>
+
     <defaultCache 
+    	maxEntriesLocalHeap="100000"
         eternal="false" 
         timeToIdleSeconds="86400" 
```

Comment:
The documentation states its possible [link](http://www.ehcache.org/generated/2.10.2/html/ehc-all/#page/Ehcache_Documentation_Set%2Fco-cfgbasics_dynamically_changing_cache_config.html%23) but it doesn't work for me. Its possibly because we are using JCache provider.

What does work for me is that at deploy time its possible to modify 
```
/var/lib/tomcat/webapps/candlepin/WEB-INF/classes/ehcache.xml
```

Would be good enough to document this on cp.org?  If not I will investigate more how to externalize the ehcache.xml or why dynamic cache config doesn't have effect at runtime.